mantle/rbx_mantle/src/config.rs

```diff
@@ -631,6 +631,7 @@ pub enum PlayableDeviceTargetConfig {
     Phone,
     Tablet,
     Console,
+    VR,

```

Comment:
Currently, serde is expecting inputs for this enum to be `vR` which is awkward. Let's add a `#[serde(rename = "vr")]` to fix that.

Reply:
Did a quick research on the internet. Should this simply be placed above vr enum? (Sorry Im not so proficient in rust so I prefer to ask in advance)

Reply:
No problem! Yup, it would be placed above `VR` like this:

```rust
pub enum PlayableDeviceTargetConfig {
    Computer,
    Phone,
    Tablet,
    Console,
    #[serde(rename = "vr")]
    VR,
}
```

Reply:
Thanks for confirmation! I now see how that works. Always something new to learn!

Should now be ok!